app/client/src/pages/logout.tsx

```diff
@@ -32,23 +32,11 @@ export default function Logout() {
                 onCompleted() {
                     resetEnv();
                     setUser(null);
-                    router.push('/login');
+                    router.push('/');
                 },
             });
         }
     }, [runMutation, isClient, resetEnv, setUser, router]);
 
-    React.useEffect(() => {
```

Comment:
Are we sure this is unnecessary now?  I don't remember what this was doing exactly or why, but it looks like it's trying to handle some sort of race condition/add a delay to logging out?

Reply:
Yeah it was a old attempt at a fix that wasn't actually necessary since going to the /logout route just directs them to '/' after running the mutation. And now I updated the logout button on the app bar to use the new useLogout hook to fire off the mutation and allow for more control over conditional behavior for onCompleted. I'm not sure we need a /logout route at all actually, I might remove it later unless you think we should keep it for some reason.

Reply:
Hmm, have you tested what logging out is like when the server is under load or with a simulated slow connection on chrome?  Logging out isn't instant and requires an async request (just like any other request).  So having a route that displays a loader until it's fully done (server responds with 200) makes sense imo.

Reply:
Ah, wait I misunderstood.  You can disregard what I said.